src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = props => {
+  const { todo } = props;
+
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {todo.user && <UserInfo user={todo.user} />}
```

Comment:
Ensure that `todo.user` is always defined before rendering the `UserInfo` component. You might want to add a default value or a fallback UI in case `todo.user` is undefined to prevent potential runtime errors.